### ISSUE TITLE
macOS client requires at least one tracker

### ISSUE THREAD

**Issue by HexomediaVR** [transmission/transmission#2361]
Hello,

I think I found a bug. I'm on Nightly Build #7489 for macOS. I add added a torrent file which has only one tracker. I can't delete that tracker if I don't add a second one. More generally, I can't have zero tracker.

Thanks for your work. :)

**Comment by fxcoudert** [comment]
I think you actually can remove all trackers. But the view is not updated once you do that, so it looks like the last tracker is still there. If you change tab and back, you'll see that the tracker is gone:

<img width="681" alt="Capture d’écran 2022-01-22 à 14 03 38" src="https://user-images.githubusercontent.com/1980544/150639627-e2474f1e-1cf9-4add-a882-896a6dc90b52.png">

(But that view is buggy, it shouldn't have such a big white space)

**Comment by fxcoudert** [comment]
When you remove the last tracker, you get:

```
*** Assertion failure in -[Torrent removeTrackers:], Torrent.mm:786
[General] Removing tracker addresses failed
```

I'm trying to follow the logic in InfoTrackersViewController.mm but I can't. Even when there is only one tracker displayed, `fTrackers.count` is equal to 2 😕

**Comment by ckerr** [comment]
@fxcoudert did #2642 solve this and/or make it easier to track down?

**Comment by sweetppro** [comment]
this seems to be fixed in the latest Build #8057
